source/_includes/head.html

```diff
@@ -23,6 +25,13 @@
   <link rel="stylesheet" href="{{ "components/patternfly/dist/css/patternfly-additions.min.css" | prepend: site.baseurl }}">
   <link rel="stylesheet" href="{{ "assets/css/patternfly-site.min.css" | prepend: site.baseurl }}">
   {% if page.url-css-extra %}<link href="{{ page.url-css-extra | prepend: site.baseurl }}" rel="stylesheet" media="screen, print">{% endif %}
+
```

Comment:
Can you move your additions up before line 27, please?  And kill the extra lines before/after.